### ISSUE TITLE
Issues when hosting

### ISSUE THREAD

**Issue by AdamPrendergast** [sbstjn/noduino#8]
I'm trying to host a simple noduino led on/off project on nodjitsu.

Serialport is creating a number of issues and not installing. Have you successfully hosted a project without this issue?

**Comment by j-5-s** [comment]
I'm new to arduino but I was under the impression that node had to be running on the same machine the arduino is connected to. Otherwise, how would node be sending and receiving data?

**Comment by AdamPrendergast** [comment]
You are completely right. In all the excitement I thought this could all be don't magically over the Internet. Obviously browsers would never allow for serialport comms.

I have managed to get it to work by utilising the a custom Cosm api. Noduino running locally subscribes to the feed and watches for changes that are triggered by a web based interface.
